health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/SearchFileIds.js

```
@@ -0,0 +1,73 @@
+      body: {
+        tenantId: Digit.ULBService.getCurrentTenantId(),
+        pageSize: "100",
+        uuid: uuids
+      },
+    });
+
+    // debugger;
+    if (response1?.user?.length === 0) {
+      throw new Error("No users found with the given uuid");
+    }
+    let uuidName={};
```

Biome

[error] This let declares a variable that is only assigned once.

'uuidName' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)